src/routes/scheduled.ts

```diff
@@ -199,11 +200,9 @@ router.post("/avtalegiro/retry", authMiddleware.isAdmin, async (req, res, next)
       /**
        * Get active agreements
        */
-      let agreements = [];
+      let agreements = await DAO.avtalegiroagreements.getByPaymentDate(claimDate.day);
```

Comment:
Fixes a potential bug in scheduled for the retry logic. This was already how it works in the "non-retry" version.

It makes sure that if the claim date is also the last day of the month (which is only possible the 28th of february), we still send a claim.

That is, make sure that if claimDate is 28th of february, we BOTH charge those who have setup the 28th, AND those who have the last day of the month.